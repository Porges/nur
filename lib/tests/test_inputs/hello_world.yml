---
version: "1.0"

default:
  run:
  - echo 'Hello, world!'
