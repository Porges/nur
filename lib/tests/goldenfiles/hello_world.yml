---
version: "1.0"

default:
  cmds:
  - echo 'Hello, world!'
